### PR TITLE
add process around Contributor License Agreements

### PR DESCRIPTION
**Pull request by afeld**
Also cleaned up some formatting. Wait to merge until someone with legal knowledge reviews.

### CODE REVIEW

**Comment by afeld** [comment]
Alternatively, one project said (via @cnelson [in Slack](https://gsa-tts.slack.com/archives/C0ENP71UG/p1482341363002023)):

> we can advise our project teams to accept your pull requests without CLA acceptance as long as you clearly mark your contributions with a public domain dedication.

Maybe we suggest this approach first?

File: practice.md [hunk 1]
```diff
@@ -29,8 +29,23 @@ As [the Free Software Foundation says](https://www.gnu.org/licenses/gpl-faq.html
...
+1. Send the agreement to GSA's Office of General Counsel (OGC) for review.
+    * Ask in [#legalstuff](https://gsa-tts.slack.com/messages/legalstuff) for who the best contact is.
+1. Collect names/emails/GitHub usernames (whatever is needed) for folks you think will be contributing.
+    * Usually easier to add too many than too few.
+1. Get it signed.
+1. Contribute.
+
+18F currently has the following CLAs signed:
+
+* [Cloud Foundry Foundation](https://www.cloudfoundry.org/governance/#docs) ([revised](https://docs.google.com/document/d/10HQY0X6SlyryhPpUrHsYtr9xYH7JYRAPk_5Ign2Du5E/edit))
```
Comment (afeld) [inline practice.md]: I think we _might_ have them for Altoros and Pivotal as well, but I'm not positive. Not sure where to look.

File: practice.md [hunk 2]
```diff
@@ -29,8 +29,23 @@ As [the Free Software Foundation says](https://www.gnu.org/licenses/gpl-faq.html
 
 > Yes. If the improvements are written by US government employees in the course of their employment, then the improvements are in the public domain. However, the improved version, as a whole, is still covered by the GNU GPL. There is no problem in this situation.
 
+### Contributor License Agreements (CLAs)
 
-#### How to license 18F repos
+Some external projects have CLAs. You cannot sign these yourself, in your official capacity.
+
+1. See if there is an organizational CLA available.
```
Comment (timothy-spencer) [inline practice.md]: How do we see this, besides the list below?  Are the channels to ask in, or is this the canonical spot?

File: practice.md [hunk 3]
```diff
@@ -29,8 +29,23 @@ As [the Free Software Foundation says](https://www.gnu.org/licenses/gpl-faq.html
...
+1. Send the agreement to GSA's Office of General Counsel (OGC) for review.
+    * Ask in [#legalstuff](https://gsa-tts.slack.com/messages/legalstuff) for who the best contact is.
+1. Collect names/emails/GitHub usernames (whatever is needed) for folks you think will be contributing.
+    * Usually easier to add too many than too few.
+1. Get it signed.
```
Comment (timothy-spencer) [inline practice.md]: Add a step here like "add the CLA to the list below"?

File: practice.md (hunk 1 above)
Comment (afeld) [reply practice.md]: @mogul Do you know?

**Comment by wslack** [comment]
@afeld I'm good with this - however, am not completely sure about the CLA piece if this is the first place its been written.

**Comment by afeld** [comment]
Re-reading this, it's less objectionable than I thought. It's the process I've followed a couple of times, and "consult OGC" is a pretty safe bet regardless. Merging. Thanks!

**Pull request merged**